cloud-sql/postgres/activerecord/config/database_tcp.yml

```diff
@@ -33,42 +33,29 @@ development:
   # but provides no arguments for configuration.
   # [END cloud_sql_postgres_activerecord_backoff]
 
-  # [START cloud_sql_postgres_activerecord_create_tcp]
+  # [START cloud_sql_postgres_activerecord_connect_tcp]
+  # Note: Saving credentials in environment variables is convenient, but not
+  # secure - consider a more secure solution such as
+  # Cloud Secret Manager (https://cloud.google.com/secret-manager) to help
+  # keep secrets safe.
   username: <%= ENV["DB_USER"] %>  # e.g. "my-database-user"
   password: <%= ENV["DB_PASS"] %> # e.g. "my-database-password"
   database: <%= ENV.fetch("DB_NAME") { "vote_development" } %>
-  host: <%= ENV.fetch("DB_HOST") { "127.0.0.1" }%> # '172.17.0.1' if deployed to GAE Flex
-  port: <%= ENV.fetch("DB_PORT")  { 5432 }%>
-# [END cloud_sql_postgres_activerecord_create_tcp]
+  host: <%= ENV.fetch("INSTANCE_HOST") { "127.0.0.1" }%> # '172.17.0.1' if deployed to GAE Flex
+  port: <%= ENV.fetch("DB_PORT") { 5432 }%>
+  # [END cloud_sql_postgres_activerecord_connect_tcp]
```

Comment:
Outdent so it is at the same level as the corresponding `START` tag.

Reply:
Done. This is actually the second occurrence of the region tag which is properly aligned with it's starting tag. I did outdent the END region tag to match the alignment the first occurrence pair of this region tag.

Reply:
Ah, I missed that. Thanks for checking.